src/app/reports/reports.service.js

```
@@ -6,23 +6,38 @@ angular.module('reports')
     dbService,
     deliveryRoundService,
     locationService,
-    pouchUtil
+    pouchUtil,
+    userStateService
   ) {
     var _this = this
 
     this.getDeliveryRounds = function (options) {
       options = options || {}
       options.limit = options.limit || 10
-      return dbService.getView('reports/delivery-rounds', options)
-        .then(function (response) {
-          return {
-            total: response.total_rows,
-            offset: response.offset,
-            results: pouchUtil.pluckValues(response)
-          }
+      return userStateService.getUserSelectedState()
+        .then(function (state) {
+          options.startkey = [state]
+          options.endkey = [state, {}]
+
+          return $q.all([dbService.getView('reports/delivery-rounds', options), _this.getRoundsCount(state)])
+            .then(function (response) {
+              return {
+                total: response[1].rows.length > 0 ? response[1].rows[0].value : 0,
+                offset: response.offset,
+                results: pouchUtil.pluckValues(response[0])
+              }
+            }).catch(function () {})
```

Why silence the error?